registration/forms.py

```diff
@@ -39,6 +40,13 @@ class Meta:
         model = User
         fields = (UsernameField(), "email")
 
+    def clean_username(self):
+        username = self.cleaned_data.get('username').lower()
```

Comment:
this should return an empty string by default instead of `None`:

`username = self.cleaned_data.get('username', '').lower()`

https://docs.python.org/3/library/stdtypes.html#dict.get